app/src/main/java/vn/edu/usth/minigh/fragments/RepoSummaryFragment.kt

```diff
@@ -0,0 +1,67 @@
+package vn.edu.usth.minigh.fragments
+
+import kotlinx.coroutines.launch
+
+import android.os.Bundle
+import android.util.Base64
+import android.util.Log
+import android.view.View
+import android.view.View.GONE
+import android.widget.LinearLayout
+import android.widget.TextView
+import android.widget.LinearLayout.LayoutParams
+import android.widget.LinearLayout.LayoutParams.MATCH_PARENT
+import android.widget.LinearLayout.LayoutParams.WRAP_CONTENT
+
+import androidx.fragment.app.Fragment
+import androidx.lifecycle.lifecycleScope
+
+import io.noties.markwon.Markwon
+
+import vn.edu.usth.minigh.R
+import vn.edu.usth.minigh.api.github
+import vn.edu.usth.minigh.api.MinRepo
+
+class RepoSummaryFragment(val repo_name: String) : Fragment(R.layout.fragment_repo_summary) {
```

Comment:
From [the docs](https://developer.android.com/reference/android/app/Fragment):

> All subclasses of Fragment must include a public no-argument constructor. The framework will often re-instantiate a fragment class when needed, in particular during state restore, and needs to be able to find this constructor to instantiate it. If the no-argument constructor is not available, a runtime exception will occur in some cases during state restore.

I suppose you want to use bundle to pass the arguments here (there's bundleOf FYI).